test/DynamoCoreWpfTests/CrashReportingTests.cs

```diff
@@ -80,7 +85,7 @@ public void StackTraceIncludedInReport()
             var dynamoVersion = AssemblyHelper.GetDynamoVersion().ToString();
 
             // Create a crash report to submit
-            var crashReport = Wpf.Utilities.CrashUtilities.BuildMarkdownContent(dynamoVersion, StackTrace);
+            var crashReport = Wpf.Utilities.CrashUtilities.BuildMarkdownContent(dynamoVersion, Packages);
```

Comment:
I would assert below that  `decoded` contains package info. And also make a new test that covers the default state of the API

Reply:
Did the change suggested on the test, but can you expand on the other test you where suggesting? The API should work the same as before the changes with the exception of it including more information (which is being checked on that test you linked). Are you referring to the method `PackagesToMakrdown(PackageLoader packageLoader)` in CrashUtilities.cs?

Reply:
yes. I would like a test covering the default state so we can assert it defaults to `No loaded packages were found.`